CHANGELOG_PENDING.md

```diff
@@ -26,4 +26,6 @@ Friendly reminder, we have a [bug bounty program](https://hackerone.com/tendermi
 
 ### BUG FIXES
 
-- [types] /#97 Fixes a typo that causes the row roots of the datasquare to be included in the DataAvailabilty header twice. (@evan-forbes)
\ No newline at end of file
+- [types] /#97 Fixes a typo that causes the row roots of the datasquare to be included in the DataAvailabilty header twice. (@evan-forbes)
+
+- [types] /#114 Fixes a typo to map the length of row roots and column roots to correct variables and mitigate confusion. (@raneet10)
```

Comment:
Note to myself: we should create a separate file with changelog entries for lazyledger specific changes. It will also help us track the modifications we made to tendermint.